frontend-react/src/components/DataDashboard/FacilityProviderSubmitterDetails/FacilityProviderSubmitterSummary.module.scss

```diff
@@ -0,0 +1,14 @@
+.FacilityProviderSubmitterSummary {
+    button, button:hover {
```

Comment:
Instead of using element selectors, each of these (`button` and `hr`) should each have their own discrete class, ie,

```
.FacilityProviderSubmitterSummary {
  &__Button {
  //styles
  }
}
```

and we can use it like: `classNames: {styles.FacilityProviderSubmitterSummary__Button}`